backend/inspirehep/records/marshmallow/base.py

```diff
@@ -38,6 +38,7 @@ class EnvelopeSchema(RecordSchemaJSONV1):
     """
 
     uuid = fields.String(dump_only=True, attribute="pid.object_uuid")
+    revision_id = fields.Integer(dump_only=True, attribute="metadata.revision_id")
```

Comment:
could you please also update the docstring?

Reply:
:+1:

Reply:
Wait. From what I can see docstring is about what Invenio is returning not for what we are returning here... I'm confused.

Reply:
we're extending `RecordSchema` from invenio